service/controller/v19/adapter/guest_internet_gateway.go

```diff
@@ -13,8 +13,9 @@ type GuestInternetGatewayAdapter struct {
 
 func (a *GuestInternetGatewayAdapter) Adapt(cfg Config) error {
 	a.ClusterID = key.ClusterID(cfg.CustomObject)
+	a.PrivateRouteTables = []string{"PrivateRouteTable"}
```

Comment:
Please add the comment from the PR description why we do it like that.

Reply:
What would be suitable place for it? I'm not sure if it makes sense to spread it everywhere or do you think copying summary-version: `Non-indexed resource name for first AZ to be compatible with old clusters.` is good enough?

Reply:
Added comments.